app/scripts/keyring-controller.js

```diff
@@ -173,10 +173,15 @@ module.exports = class KeyringController extends EventEmitter {
     })
   }
 
-  placeSeedWords () {
+  placeSeedWords (cb) {
     const firstKeyring = this.keyrings[0]
     const seedWords = firstKeyring.serialize().mnemonic
     this.configManager.setSeedWords(seedWords)
+
+    if (cb && typeof cb === 'function') {
```

Comment:
We don't need to emit an update in the other case. We already emit the update after this call, after some other changes.

I guess we don't have to type check. Seems like safer coding, but I guess it doesn't matter.